projects/schematics/src/migrations/2_0/constructor-deprecations/data/cart-coupon.component.migration.ts

```diff
@@ -99,4 +103,45 @@ export const CART_COUPON_COMPONENT_MIGRATIONS: ConstructorDeprecation[] = [
       },
     ],
   },
+  {
+    class: CART_COUPON_COMPONENT,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    deprecatedParams: [
+      {
+        className: ACTIVE_CART_SERVICE,
```

Comment:
ActiveCartService should be at the end of this list, as it is the last parameter.